kotest-assertions/kotest-assertions-json/src/commonMain/kotlin/io/kotest/assertions/json/nodes.kt

```diff
@@ -1,5 +1,8 @@
 package io.kotest.assertions.json
 
+import kotlinx.serialization.Serializable
+
+@Serializable
```

Comment:
I believe this shouldn't be required? You can try removing it in next PR perhaps :)

Reply:
I thought I've deleted it